src/cli/backup/backup.go

```diff
@@ -53,6 +54,11 @@ func AddCommands(cmd *cobra.Command) {
 		for _, addBackupTo := range serviceCommands {
 			addBackupTo(subCommand)
 		}
+
+		// TODO(meain): remove this once we stabilize group/teams backups
+		if len(os.Getenv("CORSO_ENABLE_EXPERIMENTAL_GROUP_BACKUP")) != 0 {
+			addGroupCommands(subCommand)
```

Comment:
I think this can be handled more easily by [marking the commands hidden](https://github.com/alcionai/corso/pull/4054/files#diff-27502be5c839394a5a70227e608c3dd34906db5072fdc762dc34f5292e9fccb9R53).

Reply:
~I was aware of that, but since the implementation for groups backup was "incorrect", I thought it would be better if the user didn't run the command by guessing its existence.~
As I typed this out, I realized I was being paranoid. 😆

Reply:
Hah, nothing wrong with the extra safety.  Also, I've moved the commands from `Hidden` to `PreRelease`.  So if a user does go looking for hidden commands in the code and finds these, they'll get a bit warning message that it's in progress and will probably break.